### PR TITLE
SPM-4103 add publishing config

### PR DESCRIPTION
**Pull request by robert-gurol-signavio**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,30 +1,44 @@
...
+}
+
 apply plugin: 'groovy'
 apply plugin: 'maven-publish'
 apply plugin: 'maven'
 apply plugin: 'java-gradle-plugin'
 
 group = 'net.uberfoo.gradle'
-version = '2.6-SNAPSHOT'
+version = '2.6-sig1-SNAPSHOT'
```
Comment (robert-gurol-signavio) [inline build.gradle:11]: This is still a Snapshot, I fixed everything so we can release from a local machine. 

` @Reviewer`:
**TO BE DISCUSSED: How to wire up publishing with CI.**

Comment (robert-gurol-signavio) [reply build.gradle:11]: This bit could be a separate PR / Jira issue, though.

**Review by TommyNeubert — APPROVED** [review]
Looks good to me, however iam not a gradle pro :(

File: build.gradle (hunk 1 above)
Comment (robert-gurol-signavio) [reply build.gradle:11]: Discussed with Tommy: separate PR

**Pull request merged**